bin/sl-meshctl.js

```diff
@@ -223,6 +224,7 @@ function printServiceStatus(service) {
         proc.displayId,
         proc.pid,
         proc.workerId,
+        _.pluck(proc.listeningSockets, 'port').join(','),
```

Comment:
Will this properly ignore unix domain and UDP sockets? unix domain won't have a port, not sure if pluck will return undefined or ignore that one, but UDP will have a port, I  think.